ocaml/database/db_cache_types.ml

```diff
@@ -350,10 +350,10 @@ let remove_from_set key t =
   Schema.Value.Set (List.filter (fun x -> x <> key) t)
 
 exception Duplicate
-let add_to_map key value t =
+let add_to_map idempotent key value t =
```

Comment:
I'd consider  giving`idempotent` a label because at a call site `true`/`false` isn't very descriptive and easy to mix up. You could even provide a default.